recipes/calculix/bld.bat

```diff
@@ -0,0 +1,19 @@
+cd ccx*/src
+
+rm Makefile_MT
+CP %RECIPE_DIR%\Makefile_MT Makefile_MT
+CP %BUILD_PREFIX%\Library\mingw-w64\bin\mingw32-make %BUILD_PREFIX%\Library\mingw-w64\bin\make
+
+
+REM this line translates the windows-paths to paths understandable for the mingw env
+REM -m, --mixed           like --windows, but with regular slashes (C:/WINNT)
+FOR /F "delims=\" %%i IN ('cygpath.exe -m "%LIBRARY_PREFIX%"') DO set "LIBRARY_PREFIX=%%i"
+
+make -f Makefile_MT ^
+    SPOOLES_INCLUDE_DIR="%LIBRARY_PREFIX%/mingw-w64/include/spooles" ^
+    LIB_DIR="%LIBRARY_PREFIX%/mingw-w64/lib" ^
+    FC="gfortran"
```

Comment:
@isuruf it doesn't work without setting the fortran compiler explicitly. If I do not set this f77 is used which is not available. Is this a problem with the mingw environment?